sdk/aqueduct/models/artifact.py

```diff
@@ -8,3 +8,4 @@ class ArtifactMetadata(BaseModel):
     id: uuid.UUID
     name: str
     type: ArtifactType
+    from_local_data: bool = False
```

Comment:
You need to exclude this from the json config. See the reference PR I noted in my previous comment.